Avatar/avatar.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar</title>
+    <link rel="stylesheet" href="../utility.css" />
+    <link rel="stylesheet" href="avatar.css" />
+  </head>
+  <body>
+    <div class="example-div">
+      <!-- PROFILE AVATAR -->
+      <img src="../images/avatar2.jpg" alt="avatar" class="avatar" />
+      <!-- SIZES AVATAR -->
+
+      <img
+        src="../images/avatar2.jpg"
+        alt="avatar"
+        class="avatar avatar-small"
+      />
+      <img
+        src="../images/avatar2.jpg"
+        alt="avatar"
+        class="avatar avatar-medium"
+      />
+      <img
+        src="../images/avatar2.jpg"
+        alt="avatar"
+        class="avatar avatar-large"
+      />
+      <!-- Square-AVATAR -->
+
+      <img
+        src="../images/avatar2.jpg"
+        alt=""
```

Comment:
**Fix:** I would suggest giving value to the "alt" attribute as that helps for accessibility.